pkg/storage/storagewrappers/caching.go

```diff
@@ -45,11 +45,12 @@ func (c *cachedOpenFGADatastore) ReadAuthorizationModel(ctx context.Context, sto
 		return nil, err
 	}
 
-	c.cache.Set(cacheKey, model, ttl) // these are immutable, once created, there cannot be edits, therefore they can be cached without ttl
+	c.cache.Set(cacheKey, model, ttl) // These are immutable, once created, there cannot be edits, therefore they can be cached without ttl.
```

Comment:
I feel like this comment shouldn't be buried here, it should be part of the doc of `NewCachedOpenFGADatastore`:

```
// NewOpenFGADatastoreWithCachingOfModels returns a wrapper over a datastore that caches up to maxSize *openfgav1.AuthorizationModel
// on every call to storage.ReadAuthorizationModel. It caches with unlimited TTL because models are immutable.
// It uses LRU for eviction.
```

If you want, it'd be nice if you send a follow-up PR to update the constructor name to 

```NewOpenFGADatastoreWithCachingOfModels ```

to make it clear that it only caches models, not tuples or anything else

Reply:
Hmm... yes 😬 i don't know how many people we would break because it's not like an API change, but yes.

Reply:
Considering we're already on a v1 stable, let's not add any breaking changes in minor or patch releases :) we can make use of TODOs within the codebase or compile a list of vNext things we'd like to have. Wdyt?

Reply:
Sounds good!